public/usage-examples/geometry/circle_radius/circle_radius-1-simple-oop.cs

```diff
@@ -0,0 +1,36 @@
+using SplashKitSDK;
+
+namespace CircleRadius
+{
+    public class Program
+    {
+        public static void Main()
+        {
+        SplashKit.OpenWindow("Circle Radius", 800, 600);
+        SplashKit.ClearScreen();
+
+        // Set position for the circle
+        double x_position = 400;
+        double y_position = 300;
+
+         //Create a circle A at the position (x_position, y_position)
+        Circle A = SplashKit.CircleAt(x_position, y_position, 200);
+        
+        // Find the radius of the circle A
+        double radius = SplashKit.CircleRadius(A);
```

Comment:
You could make this more OO by using `A.Radius` instead of `SplashKit.CircleRadius(A)`